File: client/src/app/_dtos/admin/orderAssignmentDto.ts
export interface IOrderAssignmentDto
{
     orderId: number;
     orderNo: number;
     orderDate: Date;
     cityOfWorking: string;
     projectManagerId: number;
     projectManagerPosition: string;
     orderItemId: number;
     hrExecId: number;
     categoryRef: string;
     categoryName: string;
     categoryId: number;
     customerId: number;
     customerName: string;
     quantity: number;
     completeBy: Date;
     postTaskAction: number;
}

export class orderAssignmentDto implements IOrderAssignmentDto
{
     orderId= 0;
     orderNo= 0;
     orderDate = new Date('1900-01-01');
     cityOfWorking= '';
     projectManagerId= 0;
     projectManagerPosition= '';
     orderItemId= 0;
     hrExecId= 0;
     categoryRef= '';
     categoryName= '';
     categoryId= 0;
     customerId= 0;
     customerName= '';
     quantity= 0;
     completeBy = new Date('1900-01-01');
     postTaskAction= 0;
}

File: api/DTOs/Admin/Orders/OrderAssignmentDto.cs
namespace api.DTOs.Admin.Orders
{
    public class OrderAssignmentDto
    {
        public int OrderId { get; set; }
        public int OrderNo { get; set; }
        public DateTime OrderDate { get; set; }
        public string CityOfWorking { get; set; }
        public int ProjectManagerId { get; set; }
        public string ProjectManagerPosition { get; set; }
        public int OrderItemId { get; set; }
        public string HrExecUsername { get; set; }
        public string CategoryRef { get; set; }
        public int ProfessionId {get; set;}
        public string ProfessionName { get; set; }
        public int CustomerId { get; set; }
        public string CustomerName { get; set; }
        public int Quantity { get; set; }
        public DateTime CompleteBy { get; set; }
       
    }
}